### PR TITLE
ts_query_ls 1.2.4 (new formula)

### PR DESCRIPTION
**Pull request by ribru17**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/ts_query_ls.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_sequoia: "0000000000000000000000000000000000000000000000000000000000000000"
+    sha256 cellar: :any_skip_relocation, arm64_sonoma:  "0000000000000000000000000000000000000000000000000000000000000000"
+    sha256 cellar: :any_skip_relocation, arm64_ventura: "0000000000000000000000000000000000000000000000000000000000000000"
+    sha256 cellar: :any_skip_relocation, sonoma:        "0000000000000000000000000000000000000000000000000000000000000000"
+    sha256 cellar: :any_skip_relocation, ventura:       "0000000000000000000000000000000000000000000000000000000000000000"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "0000000000000000000000000000000000000000000000000000000000000000"
+  end
+
```
Comment (chenrui333) [inline Formula/t/ts_query_ls.rb:17]:
```suggestion
```

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by ribru17** [comment]
Would you mind reconsidering this? I have come back with some more stars :)

**Comment by SMillerDev** [comment]
> We frown on authors submitting their own work unless it is very popular.

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff:~:text=We%20frown%20on%20authors%20submitting%20their%20own%20work%20unless%20it%20is%20very%20popular.

---

I don't think just meeting the requirements for stars is quite enough to clear that hurdle.

**Comment by clason** [comment]
Why have the requirements then? You're free to set your own, of course, but then going "lol jk" is not a good look...

**Comment by SMillerDev** [comment]
> but then going "lol jk"

The requirements are clearly documented on https://docs.brew.sh. My message didn't say "with more stars it would be accepted" it just said "whith the current popularity we can't accept it".
Just because people didn't read the requirements for inclusion and another blocker came up doesn't mean that the maintainers went "lol, jk"

**Comment by clason** [comment]
You might wish to check the feedback on your "notoriety" CI then...

**Pull request closed without merging**